source/ShipInfoPanel.cpp

```diff
@@ -390,10 +390,20 @@ void ShipInfoPanel::DrawOutfits(const Rectangle &bounds, Rectangle &cargoBounds)
 	for(const auto &cat : GameData::GetCategory(CategoryType::OUTFIT))
 	{
 		const string &category = cat.Name();
+		if(category.empty())
+			continue;
 		auto it = outfits.find(category);
 		if(it == outfits.end())
 			continue;
 
+		vector<const Outfit *> validOutfits;
```

Comment:
You could use a `std::ranges::filter_view` here instead of making a new vector. You can also use `std::ranges::all_of` to perform the check on line 404.

Reply:
Oh yeah I always forget about them